### PR TITLE
Centrality

### PR DESCRIPTION
**Pull request by mahdi-robbani**
Add

### CODE REVIEW

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:91]: `**kwargs` can lead to unexpected behaviours because you have no control over them. It is better to explicit each argument passed to the function. For those arguments that need to be passed directly to a `networkx` function, use the same name they have in the `networkx` function. Check and modify all the other functions accepting `**kwargs`.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:130]: Move the beginning of the description in the same line where the leading `"""` is, as done for the other functions' descriptions.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:137]: Leave only a blank line between functions, not two.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:189]: We can think of having the table by `pandas` since it is way tidier and it is the standard for data frames, but that would add a dependency to PyInteraph. What do you think @mtiberti and @elenapapaleo?

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:222]: You can import this function from `graph_analysis` if it is already there. The same is true for graph building. It is better not to have duplicate code, and shared functions will eventually go in a separate script anyway.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:383]: Add a black space at the end of this string, otherwise, it will merge with the first word of the next line.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:372]: Add a black space at the end of this string, otherwise, it will merge with the first word of the next line.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:447]: Add a black space at the end of this string, otherwise, it will merge with the first word of the next line.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:448]: Move the black space on the previous line.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:458]: Remove the extra blank space at the beginning of this string.

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:542]: You can repeat here in the error string what are the valid values (formatting the choices' list, not rewriting them one by one because it would be difficult to track in case they change).

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:561]: Can be removed if you pass them directly to the function as arguments.

**Review by ValeSora — CHANGES_REQUESTED** [review]
Comments regard mostly:

* duplicated code
* use of **kwargs in the functions
* typos in comments/error strings

File: pyinteraph/centrality_analysis.py
Comment (mahdi-robbani) [reply pyinteraph/centrality_analysis.py:91]: I'm not sure I understand what kind of unexpected behaviors can be caused by `**kwargs` since I only call these functions with a specific dictionary of arguments and I manually choose which arguments are in that dictionary. 
The idea behind using `**kwargs` was to make the `get_centrality_dict` function future proof. So if we ever need to add any new centrality measures (networkX or custom) then we can simply write the function separately, and just add its name to the function map and it will work regardless of what arguments it might require. Without it, the `get_centrality_dict`  will get longer with every new centrality measure since we have to perform a check to see the name of the measure and then call it from the function map using the arguments specific for that measure. 
Is the issue that if a function has `**kwargs` it can accept multiple arguments? If that is the case would it be okay if I just added a single argument for an argument dictionary, and then depending on the function it would only choose the arguments it requires from that dictionary?

File: pyinteraph/centrality_analysis.py
Comment (mahdi-robbani) [reply pyinteraph/centrality_analysis.py:222]: The `build_graph` function is slightly different from the one in `graph_analysis`  but exactly the same as the one in `path_analysis`. So should I change the `build_graph` function in `graph_analysis` or wait until `path_analysis` is merged and import it from there?

File: pyinteraph/centrality_analysis.py
Comment (mahdi-robbani) [reply pyinteraph/centrality_analysis.py:93]: I changed the names of the dictionary entries to match the networkX argument names

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [reply pyinteraph/centrality_analysis.py]: Sorry, you are right! I think we can wait for `path_analysis` to be merged and then import it from there :) let's leave this comment open then

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [reply pyinteraph/centrality_analysis.py:93]: It's not really the `**kwargs` in `get_centrality dict` but in all the other functions implementing centrality measures, since leaving the `**kwargs` in them would mean having no control over what is passed to the internal calls. Maybe let's hear also @mtiberti's opinion on this :)

File: pyinteraph/centrality_analysis.py
Comment (ValeSora) [inline pyinteraph/centrality_analysis.py:111]: I think we are using double-quoted strings throughout the code, maybe we should consider changing all those single quotes to double quotes.

**Review by ValeSora — CHANGES_REQUESTED** [review]
* Minor aesthetic changes.
* Pending @mtiberti's feedback on the `**kwargs` issue in centrality functions.
* Pending the merge of the `path_analysis` branch to import the graph-building function from there.

**Pull request merged**
